src/api-sdk/github/github.ts

```diff
@@ -1,37 +1,75 @@
-import { GitHubRepos, GitHubMembers, GitHubTeams, GitHubMembersPerTeam, GitHubReposPerTeam } from './type';
+import {
+    GitHubRepos,
+    GitHubMembers,
+    GitHubTeams,
+    GitHubMembersPerTeam,
+    GitHubReposPerTeam,
+    GitHubCollaboratorsPerRepo,
+    GitHubIssueRequest
+} from './type';
+import {
+    reposMapping,
+    membersMapping,
+    teamsMapping,
+    membersPerTeamMapping,
+    reposPerTeamMapping,
+    collaboratorsPerRepoMapping
+} from './mapping';
+
 import { ApiResponse, ApiErrorResponse } from '../response';
-import { reposMapping, membersMapping, teamsMapping, membersPerTeamMapping, reposPerTeamMapping } from './mapping';
 import { HttpRequest } from '../../http-request';
 
 export class Github {
-    constructor(private readonly request: HttpRequest) {
-        /**/
+
+    constructor(private readonly request: HttpRequest) { /**/ }
+
+    // Get info data. Could be repos/members/collaborator/teams ...
+    public async getGitHubInfo(url: string): Promise<ApiResponse<any[]> | ApiErrorResponse> {
```

Comment:
Could implement a union type for `GitHubInfo` which includes all the info interfaces? Something like:

`export type GitHubInfo = GitHubRepos | GitHubTeams | GitHubMembers | GitHubCollaboratorsPerRepo`

Reply:
The returned object does not have a well-defined data type. I have not found an official `npm` `@type` module for it, and it is not directly mapped by us, which prevents the use of our internal interface

Reply:
Fair enough, leave as `any`